src/components/InputProblem.tsx

```diff
@@ -10,10 +10,12 @@ import { Mathfield } from './Mathfield'
 import { type MathfieldElement } from 'mathlive'
 import { parse, compare } from '@khanacademy/kas'
 import { ComputeEngine } from '@cortex-js/compute-engine'
+import { type Persistor } from '../lib/persistor'
 export const MAX_CHARACTER_INPUT_PROBLEM_LENGTH = 500
 
 interface InputProblemProps extends BaseProblemProps {
   comparator: string
+  persistor?: Persistor
```

Comment:
Instead of defining this here, you can add it to `BaseProblemProps` in `ProblemSetBlock.tsx` so it gets included for all of the problem types.